packages/network/src/index.ts

```diff
@@ -8,7 +8,10 @@ export interface NetworkConfig {
   url: string;
 }
 
-export interface StacksNetwork {
+export const StacksNetworks = ['mainnet', 'testnet'] as const;
+export type StacksNetworkName = typeof StacksNetworks[number];
+
```

Comment:
Repeated `StacksNetworkName | IStacksNetwork;` type, maybe we could make a reusable type of it?

```suggestion

export type StacksNetwork = StacksNetworkName | IStacksNetwork;
```

Reply:
Here, I'd like to keep the verbose type for better DX (hover, docs, etc.) — developer will be shown both types and `StacksNetworkName` auto-expands to all its string options.